### PR TITLE
v5.8.1 CHANGELOG.md

### PR DESCRIPTION
**Pull request by esethna**

### CODE REVIEW

**Comment by esethna** [comment]
@enahum @cpanato please help review

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Mattermost Push Proxy Changelog
 
+
+## 5.8.1 Release
+- Release Date: March 28, 2019
+- Compatible with all versions of Mattermost server and mobile
+
+### Combatibility
```
Comment (enahum) [inline CHANGELOG.md]: No need for this, as this was part of 5.8.0

Comment (esethna) [reply CHANGELOG.md]: I would rather have it to be safe

Comment (cpanato) [inline CHANGELOG.md]:
```suggestion
### Compatibility
```

Comment (cpanato) [reply CHANGELOG.md]: ping @esethna

**Pull request merged**